artifacts/mobileclient_crd.yaml

```diff
@@ -29,4 +29,7 @@ spec:
               pattern: '([\w-])'
             appIdentifier:
               type: string
-              pattern: '([\w-])'
\ No newline at end of file
+              pattern: '([\w-])'
+            dmzUrl:
+              type: string
+              pattern: '(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)'
```

Comment:
This update needs to be reflected here too:
https://github.com/aerogear/mobile-core/blob/master/installer/roles/create-mobile-client-crd/files/mobile-client-crd.yaml

Reply:
@philbrookes https://github.com/aerogear/mobile-core/pull/73